assets/src/apps/authoring/components/AdaptivityEditor/ActionMutateEditor.tsx

```diff
@@ -5,19 +5,19 @@ import guid from 'utils/guid';
 import { VariablePicker, OverlayPlacements } from './VariablePicker';
 
 const typeOptions = [
-  { key: 'number', text: 'Number', value: 1 },
   { key: 'string', text: 'String', value: 2 },
+  { key: 'number', text: 'Number', value: 1 },
```

Comment:
it would be good instead of the numbers to use the CapiVariableTypes enum everywhere

Reply:
good catch. fixed.